package.json

```diff
@@ -31,16 +31,16 @@
     ]
   },
   "dependencies": {
-    "@glimmer/syntax": "^0.42.0",
     "codemod-cli": "^2.0.0",
-    "prettier": "^1.18.2"
+    "ember-template-recast": "^3.1.3"
   },
   "devDependencies": {
     "coveralls": "^3.0.6",
     "eslint": "^6.5.0",
     "eslint-config-prettier": "^6.3.0",
     "eslint-plugin-prettier": "^3.1.1",
-    "jest": "^24.9.0"
+    "jest": "^24.9.0",
+    "prettier": "^1.18.2"
```

Comment:
Just confirming, this change is in this PR specifically because we will **not** be running `prettier` on the output files once we move to ember-template-recast, right?

Reply:
correct, yes

Reply:
it is moved to `devDependencies` because we still use it internally for our own JS code though